app/helpers/admin/claims_helper.rb

```diff
@@ -68,6 +68,14 @@ def check_deadline_warning(claim)
       content_tag(:strong, pluralize(days_until_check_deadline, "day"), class: "govuk-tag #{check_deadline_warning_class}")
     end
 
+    def matching_attributes(first_claim, second_claim)
```

Comment:
Looks like this helper will includes building society roll numbers that are `nil` as matches. Can we update it to exclude those?

Reply:
Good catch! Updated.